xcat-inventory/xcclient/inventory/shell.py

```diff
@@ -58,17 +58,19 @@ def do_export(self, args):
         mgr.validate_args(args, 'export')
         mgr.export_by_type(args.type, args.name, args.path, args.directory, args.format, version=args.version, exclude=args.exclude.split(','))
 
-    @shell.arg('-o', '--origin', metavar='origin', help='origin file want to be compared')
-    @shell.arg('-n', '--new', metavar='new', help='new file want to be compared')
-    @shell.arg('-t', '--type', metavar='<type>', help='file source type, supported "git", "normal", default is "normal"')
+    @shell.arg('--files', dest='files', metavar='FILE', nargs=2, help='compare the given 2 files')
+    @shell.arg('--fn', dest='fn', metavar='FILENAME', help='the filename want to show, be used with "--files"')
+    @shell.arg('--source', dest='source', metavar='FILE', nargs=1, help='compare the given inventory file with xCAT DB, default is just compare objects file contains')
+    @shell.arg('--all', dest='all', action="store_true", default=False, help='compare the given inventory file with the whole xCAT DB, be used with "--source"')
```

Comment:
should better be `help='compare the given inventory file with the whole inventory in xCAT DB, be used with "--source". If not specified, will only compare the objects in given inventory file by default.'`